README.md

```diff
@@ -1,2 +1,9 @@
 # cookie_stand
 CodeFellows 201 Cookie Stand Project
+
+day1FeatureWork
+Created individual pages and fleshing out basic structure.  Working on sales.html and app.js mostly.  
+
+salmon.png - credit to Sam Hamm, samhamm@gmail.com, https://github.com/samhamm
+
+Math.random function structure copied from: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random
```

Comment:
Nice verbose updating of the README. Keep up with that habit, it'll take you far.